webapp/channels/src/stores/local_storage_store.ts

```diff
@@ -156,6 +157,20 @@ class LocalStorageStoreClass {
     getWasLoggedIn() {
         return this.getItem(getWasLoggedInKey()) === 'true';
     }
+
+    // the following flag's setter and getter are used to make sure a user is notified (via aria-label)
+    // about a successful login only once (MM-50821)
+    setWasNotifiedOfLogIn(wasNotified: boolean) {
```

Comment:
This can be simplified to be:

```
setWasNotifiedOfLogIn(wasNotified: boolean) {
    this.setItem(wasNotifiedOfLogInKey, String(wasNotified));
}
```

Reply:
thanks, I like your suggestion, it looks more elegant 👍